### ISSUE TITLE
PNI: Hero Image and Text Contrast 

### ISSUE THREAD

**Issue by sabrinang** [mozilla/foundation.mozilla.org#5701]
Improve legibility and contrast between hero image and intro copy on PNI home page. The report was for mobile but check across viewport/screen sizes.

via slack from dholbert/janicet
<img width="393" alt="image" src="https://user-images.githubusercontent.com/2492510/99997666-57f30c80-2d72-11eb-86af-498dc2b2dfde.png">

**Comment by natalieworth** [comment]
I've decreased the contrast in the pni hero image and [uploaded new file here](https://drive.google.com/open?id=1P3yl51VtRrGfAS8-5d7Wf-6V9DLJzGA0&authuser=natalie%40mozillafoundation.org&usp=drive_fs).

**Comment by natalieworth** [comment]
[Redpen tests are in here](https://redpen.io/p/nc9396ac0bf1b8e29a)

**Comment by natalieworth** [comment]
I checked the darkest grey shadows against the type in the Web accessibility contrast checker. The original actually still passes, but I adjusted the contrasted of the shadows and dodged some of the very dark areas where type is in photoshop to help with readability.

I've uploaded the new designs into [the Redpen here](https://redpen.io/p/nc9396ac0bf1b8e29a).

Just need design R+ and I will make a ticket for Mavis to implement.

This is the original (with the darkest grey shadow selected for contrast check). It still passes but I feel it's somewhat difficult to read with text overlayed:
![Screen Shot 2020-12-06 at 2.22.42 PM.png](https://images.zenhubusercontent.com/57a10963e40e5714b16cdb3b/08027638-9aca-4977-a0a6-d5a4a7e5d728)


This is with the adjusted contrasted:
![Screen Shot 2020-12-06 at 2.23.40 PM.png](https://images.zenhubusercontent.com/57a10963e40e5714b16cdb3b/c0884e69-0ee9-4124-9d63-4b87e1545e7b)

**Comment by natalieworth** [comment]
Made a dev ticket for Mavis [#5823](https://github.com/mozilla/foundation.mozilla.org/issues/5823), so closing this. :)
